e2e-chatbot-app-next/README.md

```diff
@@ -52,30 +52,82 @@ This project includes a [Databricks Asset Bundle (DAB)](https://docs.databricks.
 3. **Specify serving endpoint and address TODOs in databricks.yml**: Address the TODOs in `databricks.yml`, setting the default value of `serving_endpoint_name` to the name of the custom code agent or Agent Bricks endpoint to chat with.
    - NOTE: if using [Agent Bricks Multi-Agent Supervisor](https://docs.databricks.com/aws/en/generative-ai/agent-bricks/multi-agent-supervisor), you need to additionally grant the app service principal the `CAN_QUERY` permission on the underlying agent(s) that the MAS orchestrates. You can do this by adding those
      agent serving endpoints as resources in `databricks.yml` (see the NOTE in `databricks.yml` on this)
-4. **Validate the bundle configuration**:
+4. **(Optional) Enable database integration**: By default, the app deploys in **ephemeral mode** without a database. To enable persistent chat history, you need to uncomment **TWO sections** in `databricks.yml`:
+
+   **Step 1: Uncomment the database instance resource**
+
+   Find this section (around lines 17-21):
```

Comment:
IMO it's better to have `TODO (optional): ...` in the `databricks.yml` file itself and ask folks to go through the TODOs, since the exact contents of the file may change over time and it's best not to duplicate it / reference line numbers etc in the README (just keep one source of truth in the file itself)